sor/src/main/java/com/bazaarvoice/emodb/sor/core/DefaultDataStore.java

```diff
@@ -690,6 +679,24 @@ public void beforeWrite(Collection<RecordUpdate> updateBatch) {
                     _eventBus.post(new UpdateIntentEvent(this, updateRefs));
                 }
             }
+
+            public void afterWrite(Collection<RecordUpdate> updateBatch) {
+                // Write the audit to the audit store after we know the delta has written sucessfully.
+                // Using this model for writing audits, there should never be any audit written for a delta that
+                // didn't end in Cassandra. However, it is absolutely possible for audits to be missing;
```

Comment:
Minor: This sentence makes me shiver with antici-

I assume there was going to be a list of why it's possible for audits to be missing.

-pation

Reply:
👍